src/database.cc

```diff
@@ -266,12 +266,8 @@ NAN_METHOD(Database::Close) {
           v8::Local<v8::Value> argv[] = {
               Nan::New<v8::FunctionTemplate>(EmptyMethod)->GetFunction() // empty callback
           };
-          Nan::MakeCallback(
-              iterator->handle()
-            , end
-            , 1
-            , argv
-          );
+          Nan::AsyncResource ar("leveldown:iterator.end");
+          ar.runInAsyncScope(iterator->handle(), end, 1, argv);
```

Comment:
`Nan::MakeCallback()` is deprecated. We should instead use `Nan::AsyncResource::runInAsyncScope()` which has the same api.